### DISCUSSION TITLE
Is Anura3D able to account for flexibility of structure in modelling soil behavior?

### DISCUSSION THREAD

**Discussion by HaodeW** [Anura3D/Anura3D_OpenSource#99] [Q&A] [answered]
Hi all,

I am wondering if Anura3D is able to account for the flexibility of a structure in modelling soil behavior. An example I was looking at is  the loading of a circular/strip footing. From my understanding, the moving mesh function has to be used whether I prescribe a vertical distributed load or a vertical prescribed velocity to the footing. When I apply either the distributed load/prescribed velocity to the footing, with the use of the moving mesh function, I obtain the same vertical stress-displacement relationship (right at the center of the footing and the edge of the footing) regardless whether the Young's Modulus of the footing is 30 GPa or 300 MPa. I ran the same model in OptumG2 and double checked with some existing literature and they gave me different values (vertical stress-displacement) for different Young Modulus. 

As such, i have the following two questions regarding this simple example of a footing: 
1) will the moving mesh function inherently imbues some form of rigidity into the structure?
2) how do we account for the flexibility of a structure in modelling soil behavior?

I would greatly appreciate any insights anyone may have regarding this issue. Thank you very much!

**Comment by murphy-john** [comment, marked as answer]
I asked around and the moving mesh doesn't "force" the material to be a rigid body. The moving mesh will follow the average velocities of the material points within the bounds of the moving mesh. However, the consensus is you'd run into issues if you used the moving mesh with two deformable bodies. Let's talk about this more when we meet with Joe again.
